Src/BootCamp.Chapter/ArrowMovement.cs

```diff
@@ -18,7 +18,14 @@ public static class ArrowMovement
         /// <returns>One of the arrow characters. '↥' by default.</returns>
         public static char GetIndicator(char symbol)
         {
-            return '-';
+            return char.ToLower(symbol) switch
```

Comment:
I wasn't familiar with pattern matching at all and I was reluctant to use it. Eventually, I did some reading and holy, this is really useful!

Reply:
damn right 👍